Bottom: 0c9aee56dbaecfa138715806a5905cd74993c1b4
Top:    7e9324f4bc6bed6e487a8af4f7f36cb559ab4572
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-07 21:50:58 -0700

Refresh of dbg-out

---

diff --git a/src/preload.c b/src/preload.c
index 4379bfc..be2487f 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -442,6 +442,8 @@ int accept(int socket, struct sockaddr *addr, socklen_t *addrlen)
 	enum fd_type type;
 
 	type = fd_get(socket, &fd);
+	fprintf(fout, "%d accept fd %d\n", syscall(SYS_gettid), socket);
+	fflush(fout);
 	if (type == fd_rsocket || type == fd_fork) {
 		index = fd_open();
 		if (index < 0)
@@ -454,6 +456,8 @@ int accept(int socket, struct sockaddr *addr, socklen_t *addrlen)
 			return ret;
 		}
 
+		fprintf(fout, "%d accept fd %d new fd\n", syscall(SYS_gettid), socket, index);
+		fflush(fout);
 		fd_store(index, ret, type);
 		return index;
 	} else {
